crates/bevy_sprite/src/mesh2d/material.rs

```diff
@@ -366,9 +374,151 @@ pub fn queue_material2d_meshes<M: SpecializedMaterial2d>(
     }
 }
 
-/// A component bundle for entities with a [`Mesh2dHandle`] and a [`SpecializedMaterial2d`].
+/// Data prepared for a [`Material`] instance.
```

Comment:
This is the fix for the failing doc test
```suggestion
/// Data prepared for a [`Material2d`] instance.
```

Reply:
Thanks